lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -6743,7 +6743,11 @@ size_t TypeSystemClang::GetIndexOfChildMemberWithName(
           if (field_name.empty()) {
             CompilerType field_type = GetType(field->getType());
             std::vector<uint32_t> save_indices = child_indexes;
-            child_indexes.push_back(child_idx);
+            // We have to add on the number of non-empty base classes to this
+            // index!
```

Comment:
This comment is redundant. The `non-empty base classes` is no longer true now that we're forwarding `omit_empty_base_classes`

Reply:
I will remove the comment